packages/api-key-service/src/fetch.public.keys.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs')
```

Comment:
I reckon this file should live in utils - looks like something we'll use again :-)

Reply:
Module looks good BTW 👌🏻

Reply:
Defo, I've used it for depoyments too so I shall move 👍

Reply:
sorted 👍